src/SprykerEco/Yves/Adyen/Handler/Notification/Mapper/AdyenNotificationMapper.php

```diff
@@ -71,7 +71,7 @@ protected function createNotificationsTransfer(array $response): AdyenNotificati
     /**
      * @param array $response
```

Comment:
What about this key and value types?

Reply:
It seems we can't be sure exactly for which type of values in every item, because I have the error from PHPStan about it: "Parameter #2 $input1 of function array_map expects array, string given", if I set array<string, string>, but, it doesn't mean that all values are arrays as well. I decided to leave as is.